src/js/views/patients/patient/archive/archive_views.js

```diff
@@ -267,25 +274,42 @@ const LayoutView = View.extend({
     content: {
       el: '[data-content-region]',
       regionClass: PreloadRegion,
-      replaceElement: true,
     },
   },
   template: hbs`
-    <div class="patient__tabs">
+    <div class="patient__tabs js-list-header">
       <button class="patient__tab js-dashboard">
         {{~ @intl.patients.patient.archive.archiveViews.dashboardBtn ~}}
       </button>
       <span class="patient__tab--selected">
         {{~ @intl.patients.patient.archive.archiveViews.archiveBtn ~}}
       </span>
     </div>
-    <div class="patient__list-container flex-region">
-      <div data-content-region></div>
+    <div class="patient__list-container flex-region js-list" data-content-region>
     </div>
   `,
+  ui: {
+    listHeader: '.js-list-header',
+    list: '.js-list',
+  },
   triggers: {
     'click .js-dashboard': 'click:dashboard',
   },
+  initialize() {
```

Comment:
not thrilled with having to do this.  But if we are we should make it a behavior

Reply:
I like it from a user perspective. But ya I'm not a huge fan either of the code implementation. I just don't know of another way to do it.

Reply:
I just pushed a commit with the behavior added.

Reply:
Coverage miss was on the archive views. Because I forgot the `childViewEvents: { 'update:listDom': 'fixListWidth' }`.

Which I added in the [last commit](https://github.com/RoundingWell/care-ops-frontend/pull/1461/files#diff-517afe962d13585a95fd46396c804969978a82f1d71412ed0d3ff4227dd3c1b5R301-R303). So coverage should be good now 🤞